control-plane/plugin/src/bin/rest-plugin/main.rs

```diff
@@ -116,5 +116,10 @@ async fn execute(cli_args: CliArgs) {
                 node::Node::uncordon(id, label, &cli_args.output).await
             }
         },
+        Operations::Upgrade(resource) => match resource {
```

Comment:
Right I see, so this upgrade command should not live here in the rest-plugin, only in the kubectl-plugin.